api/lib/handlers/login/index.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var config = require('../../config.js').config;
+var pg = require('pg');
+
+module.exports.post = function(req, res, next) {
+  var id = req.body.id;
```

Comment:
We should validate user inputs, not just for security reasons (in this case it might be ok) but also to display a meaningful error message to the client